src/navigator/Headers.tsx

```diff
@@ -216,7 +217,7 @@ export function HeaderTitleWithBalance({
   )
 }
 
-export function HeaderTitleWithTokenBalance({
+export function HeaderTitleWithTokenBalanceByAddress({
```

Comment:
why is this necessary? it has the same input types as the new version. When I checked out your branch and swapped the only usage of this out for HeaderTitleWithTokenBalance, no type errors resulted. Am I missing something?

Reply:
Removed in https://github.com/valora-inc/wallet/pull/4242/commits/330bbe245ffe9a8a89a2185d0cbb2ac89d9e50ae!